src/main/resources/templates/admin/include/footer.ftl

```diff
@@ -1,6 +1,9 @@
 <script>
     // prevent double form submission
     $('button').click(function () {
+        if ($('#postForm').length > 0) {
```

Comment:
use more specific id, for example, "postContentEditorForm", as long as this script is global for whole admin panel